test/e2e/test_broker_pubsub.go

```diff
@@ -61,6 +66,76 @@ func BrokerWithPubSubChannelTestImpl(t *testing.T) {
 	client := lib.Setup(t, true)
 	defer lib.TearDown(client)
 
+	u := createBrokerWithPubSubChannel(t, client, brokerName, dummyTriggerName, kserviceName, respTriggerName, targetName)
```

Comment:
Line breaks? Also for the following long lines.

Reply:
done